aiidalab_launch/profile.py

```diff
@@ -106,6 +115,7 @@ def from_container(cls, container: Container) -> Profile:
             else container.image.tags[0]
         )
 
+        # TODO: How to detect custom mounts?
```

Comment:
Have a look at the output of `docker inspect $CONTAINER_ID`. You should be able to find the additional volumes there. If not, let me know I can try to investigate further.